src/Tribe/Promoter/Observer.php

```diff
@@ -41,21 +41,25 @@ public function hook() {
 		$this->registered_types();
 
 		// Listen for changes on RSVP as Gutenberg Uses the post_type API to update RSVP's
-		add_action( 'save_post_tribe_rsvp_tickets', [ $this, 'notify_rsvp_event' ], 10, 2 );
+		add_action( 'save_post_tribe_rsvp_tickets', [ $this, 'notify_ticket_event' ], 10, 1 );
 
 		// RSVP
 		add_action( 'tickets_rsvp_ticket_deleted', [ $this, 'notify_event_id' ], 10, 2 );
 		add_action( 'event_tickets_rsvp_tickets_generated', [ $this, 'notify_event_id' ], 10, 2 );
 
+		// Moved tickets
```

Comment:
:no_entry_sign: Inline comments must end in full-stops, exclamation marks, or question marks

<sub>hash: 03384e5dd68f010a2c5d58fab069c398</sub>